src/Parser/Druid/Feral/Modules/Normalizers/RakeBleed.js

```diff
@@ -0,0 +1,45 @@
+import SPELLS from 'common/SPELLS';
+import EventsNormalizer from 'Parser/Core/EventsNormalizer';
+
+const CAST_WINDOW = 100;
+class RakeBleed extends EventsNormalizer {
```

Comment:
This looks good, but please consider adding a test for this class. Classes like Normalizers are very important to work well and so a test suite ensuring their behavior could prevent issues in the future.